tasks/rpmbuild-test

```diff
@@ -1,69 +1,86 @@
 #!/bin/bash
 set +e
 
-base_dir="$(dirname $0)/.."
+function acquire_lock {
+     # Write uuid to a lock file and store a backup
+     uuidgen > file.lock
+     cp file.lock uuid.saved
+     while true; do
+          # Check if lock exists on remote server
+          while [[ $(rsync --ignore-existing --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${fed_branch}/repo/lockdir) != *"file.lock"* ]]; do
+               sleep 60
+          done
+          cp uuid.saved file.lock
+          # Push lock file with uuid to remote server
+          rsync --ignore-existing -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${fed_branch}/repo/lockdir/
+          # Pull lock file back
+          rsync -avz fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${fed_branch}/repo/lockdir/file.lock file.lock
+          # If uuid matches, we can proceed
+          if [[ $(diff file.lock uuid.saved) == "" ]]; then
+               break
+          fi
+          sleep 60
+     done
+}
+
+# Kill backgrounded jobs on exit
+function clean_up {
```

Comment:
Yea that's the point for now.  Nothing calls acquire_lock either.  We are leaving it commented out until everything is all set with the 2.0 pipeline